src/app/pages/Kafka/KafkaMainView.tsx

```diff
@@ -17,17 +17,6 @@ export const KafkaMainView = (): React.ReactElement => {
   const kafka = useKafkaInstance();
 
   const [error, setError] = useState<undefined | number>();
-  const [isInstanceDrawerOpen, setIsInstanceDrawerOpen] = useState<boolean | undefined>();
-  const [activeDrawerTab, setActiveDrawerTab] = useState<string>('');
-
-  const handleInstanceDrawer = (isOpen: boolean, activeTab?: string) => {
-    activeTab && setActiveDrawerTab(activeTab);
-    setIsInstanceDrawerOpen(isOpen);
-  };
-
-  const onCloseInstanceDrawer = () => {
-    setIsInstanceDrawerOpen(false);
-  };
```

Comment:
What happened is that the drawer's state was handled by the drawer's context from kas-ui, so this bit of state here was doing nothing for us.

In order to keep kafka-ui decoupled from kas-ui, the federated instance component takes now a render prop that passes the helper functions to interact with the drawer itself.